src/ml/neural_net/tf_compute_context.hpp

```diff
@@ -0,0 +1,52 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef UNITY_TOOLKITS_NEURAL_NET_TF_COMPUTE_CONTEXT_HPP_
+#define UNITY_TOOLKITS_NEURAL_NET_TF_COMPUTE_CONTEXT_HPP_
+
+#include <ml/neural_net/compute_context.hpp>
+
+namespace turi {
+namespace neural_net {
+
+
+/**
+ * A compute_context implementation backed by MXNet
+ * for neural network computation and for data
+ * augmentation.
+ */
+class tf_compute_context: public compute_context {
+public:
+
+
+  /**
+   * Constructs a context wrapping the best currently available Metal device.
```

Comment:
For TensorFlow, there is no Metal device, and I don't think we will support eGPU for the foreseeable future